xrootd-storage/gratia-xrootd-storage

```diff
@@ -97,7 +100,7 @@ class XrootdSummaryParser(ContentHandler):
     def emitData(self, key, val):
         try:
             self.dataCallback(key, val)
-        except SystemExit, KeyboardInterrupt:
+        except SystemExit as KeyboardInterrupt:
```

Comment:
```suggestion
        except (SystemExit, KeyboardInterrupt) as _:
```
I think this what you _want_ to do -- catch both SystemExit and KeyboardInterrupt; in this case, the two need to be in a tuple in Python 2. (What the previous code was doing in Python 2 was catching SystemExit and storing the exception in KeyboardInterrupt.)